itests/eth_deploy_test.go

```diff
@@ -91,25 +92,22 @@ func TestDeployment(t *testing.T) {
 	pendingFilter, err := client.EthNewPendingTransactionFilter(ctx)
 	require.NoError(t, err)
 
-	// Pause so we can test that everything works while the message is in the message pool.
-	for _, miner := range miners {
-		miner.Pause()
```

Comment:
why are we disabling this? we seem to be indexing within `EthSendRawTransaction` so the pause - submit - check - unpause - wait flow should still work, no?